src/main/resources/api/data-repository-openapi.yaml

```
@@ -2510,6 +2510,11 @@ paths:
                 $ref: '#/components/schemas/DRSError'
 components:
   schemas:
+    TableDatatypes:
```

I would expect this to be a singular, not plural name. A field of this type can only be a single type at a time, so it's not a collection of types.
```suggestion
    TableDatatype:
```